src/main/java/org/mamute/list/Tab.java

```diff
@@ -0,0 +1,32 @@
+package org.mamute.list;
+
+public class Tab {
+	private final Type type;
+	private final String localizationKey;
+	private final String link;
+
+	public Tab(Type type, String localizationKey, String link)
+	{
+		this.type = type;
+		this.localizationKey = localizationKey;
+		this.link = link;
+	}
+
+	public String getLocalizationKey() { return this.localizationKey; }
+	public String getLink() { return this.link; }
+	public Type getType() { return this.type; }
+
+	public enum Type {
```

Comment:
I would make the type separated from the Tab class(it could be within the same package with package-private access modifier)
Is the Tab class really necessary here?